core/js/decanter.js

```diff
@@ -1,3 +1,595 @@
-/**
- * Empty until further notice.
- */
+// if NodeList doesn't support forEach, use Array's forEach()
+NodeList.prototype.forEach = NodeList.prototype.forEach || Array.prototype.forEach;
+
+document.addEventListener( "DOMContentLoaded", event => {
+
+  // The css class that this following behaviour is applied to.
```

Comment:
These global variables are defined here but way down in line 560 you have a code comment for global scope variables. Can we move these all to one spot?